src/braces.cpp

```diff
@@ -528,14 +558,18 @@ static void examine_brace(chunk_t *bopen)
                || chunk_is_token(pc, CT_WHILE)
                || chunk_is_token(pc, CT_SWITCH)
                || chunk_is_token(pc, CT_USING_STMT)
-               || (  chunk_is_token(pc, CT_BRACE_OPEN)
-                  && chunk_is_token(prev, CT_FPAREN_CLOSE)))
+               || chunk_is_token(pc, CT_BRACE_OPEN)) // Issue #1758
```

Comment:
Really hard to tell apart changes. Is this the only functional change?

Reply:
It is the consequence of the bug #1758 
The condition  `&& chunk_is_token(prev, CT_FPAREN_CLOSE)` was not apropriated to solve the case.

Reply:
The question is "Is this the **only** change that fixes the bug" ?

Reply:
because they are mixed with other changes, that's why I ask.

Reply:
Yes. It is the only line.
I made lot of changes by the use of the debug function `LOG_FMT(...)`
to find out what the matter is.
I might be better to make to PR for that.

Reply:
no, it's fine like this but yeah, maybe next time different commits with different things changed would help in review.

Reply:
Anybody to approve?

Reply:
Hmm that whole function is a bit strange ... here is one failure:

```cpp
int main()
{

for(int f=0; f < 1; f++)
{
	auto a = int{1};
}

return 0;
}
```